lib/msf/core/module/ui/message.rb

```diff
@@ -14,9 +14,8 @@ def print_good(msg='')
 
   def print_prefix
     prefix = ''
-    if (datastore['TimestampOutput'] =~ /^(t|y|1)/i) || (
-      framework && framework.datastore['TimestampOutput'] =~ /^(t|y|1)/i
-    )
+    if datastore['TimestampOutput'] ||
```

Comment:
Or maybe what I'm saying is, we just need to make the datastore work like Standard ML's type inference system. @acammack-r7 , you with me :) ?